dotCMS/src/main/resources/dotmarketing-config.properties

```diff
@@ -272,7 +272,7 @@ path.to.dojo=/html/js/dojo/custom-build
 URLMAP_FALLTHROUGH=true
 
 ##	Reserved Folder Names
-RESERVEDFOLDERNAMES=WEB-INF,META-INF,assets,dotcms,html,portal,email_backups,DOTLESS,DOTSASS
+RESERVEDFOLDERNAMES=WEB-INF,META-INF,assets,dotcms,html,portal,email_backups,DOTLESS,DOTSASS,dotAdmin,custom_elements
```

Comment:
I don't think it's safe to put these restrictions in the .properties file, as customers might unintentionally change it without really knowing the consequences of it. They should only be available as they are in the `FolderAPIImpl` class.

Reply:
I am ok if we remove this from property file and have the default values only as defaults in the class using the overloaded `Config.getStringProperty` methods.  This hides the config so people don't mess with it unless they know what they are doing and want to override the property, at which point that is on them.

Reply:
so @nollymar we need to remove the property from the `dotmarketing-config.properties`

Reply:
done